response_operations_ui/views/messages.py

```diff
@@ -22,10 +22,13 @@ def view_messages():
         'label': request.args.get('label'),
         'survey': request.args.get('survey')
     }
-    messages = message_controllers.get_message_list(params)
-    refined_messages = [_refine(msg) for msg in messages]
     breadcrumbs = [{"title": "Messages"}]
-    return render_template("messages.html", breadcrumbs=breadcrumbs, messages=refined_messages)
+    messages = message_controllers.get_message_list(params)
```

Comment:
I would make `get_message_list` throw a `NoMessages` that subclasses `Exception` error or a `KeyError` and catch that here, taking away the if statements.